internal/context/submission/domain/run/event_submit_handler.go

```diff
@@ -111,3 +118,17 @@ func (e *EventHandlerSubmitRun) republicCurrentEvent(ctx context.Context, event
 	}
 	return nil
 }
+
+func mergeWorkflowParams(input, output map[string]interface{}) map[string]interface{} {
```

Comment:
The `mergeWorkflowParmas` function is only used in the logic of the previous issue. Because the above logic is deleted, the function is no longer needed, so it is also deleted.

Reply:
Merging maps is commonly used, so a generic function `MergeMap` had be added in `pkg.utils` package,